.gitignore

```diff
@@ -17,4 +17,4 @@ npm-debug.log*
 pids
 .env
 .DS_Store
-assets/state.json
+assets/state.json
```

Comment:
Hmm - according to `.editorconfig` we should have a trailing newline https://github.com/redbadger/website-honestly/blob/master/.editorconfig#L6

Not sure if VScode will pick up on this file by default?